Mobile Test Runner for Jama/Mobile Test Runner for Jama/LoginViewController.swift

```diff
@@ -117,6 +118,7 @@ extension LoginViewController: EndpointDelegate{
             //Notify user of unauthorized reload the view on the main thread
             DispatchQueue.main.async {
                 self.reloadViewWithUsernameAndInstanceSaved()
+                self.viewDidLoad()
```

Comment:
Somehow this was the only way for us to fix the 'done' button issue with the keyboard. (After you log in and hit the logout/back button or attempt a log in and fail, the done button no longer hides the keyboard.)